test/e2e-kind.sh

```diff
@@ -225,7 +225,7 @@ install_certmanager() {
 
     echo 'Installing cert-manager...'
     docker_exec kubectl create namespace cert-manager
-    docker_exec kubectl create secret tls kubernetes-root-ca \
+    docker_exec kubectl create secret tls kubernetes-intermediate-ca \
```

Comment:
is this correct? Just want an explanation of this since we havent needed to change this in a while. Does KBA also need to change?

Reply:
will remove once cert-manager chart is merged